tcipostman/__init__.py

```diff
@@ -0,0 +1,25 @@
+from flask_mail import Mail, Message
+from tcidatabase import models
+from tcidatabase import db
+
+
+def init_app(app):
+    Postman.app = app
+    Postman.mail = Mail(app)
+    db.signals.post_save.connect(Postman.send_client_token,
+                                 sender=models.SingleToken)
+
+
+class Postman:
+    app = None
+    mail = None
```

Comment:
I don't get it when `app` and `mail` are correctly set up in the function `send_client_token` with a project that use Postman